.gitlab-ci.yml

```diff
@@ -49,7 +49,7 @@ test-syntax:7.0:
 ## Building the artifacts for release
 
 frontend:
-  image: node:8.9
+  image: node:8.11
```

Comment:
We could simply use node:8, since the API should stay compatible in major versions, no?

Reply:
I just align it to my dev environment. Theoretically the minor version should not change the API